platform/email/platform-email-impl/src/main/java/org/codice/ddf/platform/email/impl/SmtpClientImpl.java

```diff
@@ -136,11 +136,15 @@ public Session createSession() {
     notNull(message, "message must be non-null");
     return executorService.submit(
         () -> {
+          final ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+          Thread.currentThread().setContextClassLoader(Transport.class.getClassLoader());
```

Comment:
😲 I had no idea you could set the context classloader. I'm pretty sure this was the part I was missing; great job!

Reply:
If you change the class loader, should you reset it to the original value after the work is complete? Or does the fact that it runs in a separate thread in the executor service make that unnecessary? If it does need to be reset, then do it with a try-finally.

Reply:
I am resetting the class loader in the `finally` block.